### ISSUE TITLE
Best answer

### ISSUE THREAD

**Issue by ceyhansuyu** [tierra/topicsolved#33]
How can I do best answer and topic solved?

![topicsolved](https://cloud.githubusercontent.com/assets/2848362/9514593/0c1cd642-4ca4-11e5-82a4-e4bf84b33861.png)

**Comment by tierra** [comment]
For it's initial release, it's not very flexible in regards to customization yet. I would recommend building your own extension that adds custom CSS in order to modify the style of the indicators. I haven't done this for my own indicators, but I have created an extension for my own board to manage custom CSS styles that you can use as an example:
https://github.com/tierra/wxforum-styles

**Comment by ceyhansuyu** [comment]
Thank you.

**Comment by alekks** [comment]
I'm not sure, but my interpretation of ceyhansuyu's request isn't about custom styling/formatting but to add a feature to mark a post within a thread as "Best answer". A bit like a lot of other forums work, e.g. Apple's support forums, where you can mark posts as "This helped me" etc.

**Comment by tierra** [comment]
Well, if it's not about the style, it's about the text, and the text _can_ already be customized (from the ACP) to say exactly what's in the screenshots. Maybe you're thinking of some third thing altogether though: something like upvotes?

**Comment by Galixte** [comment]
Yes that which is particular that’s the text in the subject of the post is: “Best answer” while the text in the subject of the topic stays: “Solved”.

They want two texts, one for the topic title and one for the title of the best answer.

**Comment by tierra** [comment]
Oh, right, I do remember thinking that originally, and I think that's why I chose to recommend custom CSS.

**Comment by tripflex** [comment]
For me this was very easy to do by modifying the code .. IMO it should be as easy as just adding another setting for "Selected Answer Phrase" and then in the function `viewtopic_modify_post_row_subject` in `event/main_listener.php` just use that instead of the default solved text.

For anybody else looking to do this, just go to this line in this file:
https://github.com/tierra/topicsolved/blob/master/event/main_listener.php#L294

And add:

``` php
$topic_data['forum_solve_text'] = 'Selected Answer';
```

or whatever you want it to say.

I'm a full time WordPress PHP dev but first time using PHPBB so i'm not sure if the fact that me using a custom theme makes a difference or not, or if doing this will change it anywhere else (from my tests it only did on the view topic page), but this should get you pointed in the right direction at least.

ALSO: I don't think this will work if you're using images, so keep that in mind
